src/views/SystemList/index.tsx

```diff
@@ -64,7 +76,7 @@ export const SystemList = () => {
           </SummaryBox>
         </SectionWrapper>
 
-        {loading ? (
+        {loadingSystems || loadingBookmarks ? (
```

Comment:
I think we should separate out the conditional for loading both bookmarks and system data.  When clicking a bookmark, the table re-renders as well due to this condition.   Ideally we want only bookmarks to reload.  Separate out the Bookmark and Table sections to meet their own loading checks.